nucliadb_node/src/shards/shard_writer.rs

```diff
@@ -321,18 +320,12 @@ impl ShardWriter {
         })
     }
 
-    pub fn create_vectors_index(&self, new: NewVectorsIndex) -> NodeResult<()> {
+    pub fn create_vectors_index(&self, shard_id: String, name: String, new: VectorConfig) -> NodeResult<()> {
```

Comment:
`shard_id` is redundant, it should be obtained from `self` (or `self.metadata`)

Reply:
True! changing! :pray: